dashboard/test/ui/step_definitions/eyes_steps.rb

```diff
@@ -6,6 +6,10 @@
 
 When(/^I open my eyes to test "([^"]*)"$/) do |test_name|
   ensure_eyes_available
+
+  batch_name = test_name + " | " + ENV['BATCH_NAME']
+  @eyes.batch = Applitools::Base::BatchInfo.new(batch_name)
```

Comment:
Unless we really want to keep the tests as separate batches, and use the batch functionality to test different platforms/resolutions, as they do in the screenshot.